### ISSUE TITLE
about the times to train a ssd-resnet  model,please,help me !

### ISSUE THREAD

**Issue by zibojia** [weiliu89/caffe#927]
Is that normal that I  trained a model for 60000 times with only a 0.36 accuracy?

**Comment by HolmesShuan** [comment]
The answer is for reference only. We train SSD using `resnet-18` on K80x3 with batchsize=128. `mAP@0.5` reaches `0.37` when 80000 iterations have passed. We also encounter a vexing issue, which is `batch normalization` layer cannot be used in SSD-resnet18, otherwise, the `mbox_loss` remains **nan**.

**Comment by Yizhouw21** [comment]
@HolmesShuan Hi! Thanks for sharing your result. I am wondering which data set were you using? I am also trying to fine-tune SSD with resnet-18 on PASSCAL VOC2007 trainval, and I got 0.39mAP on VOC2007 test.

**Comment by HolmesShuan** [comment]
I also use `PASSCAL VOC2007 trainval`. It seems that [MobileNet-SSD](https://github.com/chuanqi305/MobileNet-SSD) is a better choice.
